llmfoundry/data/finetuning/tasks.py

```diff
@@ -49,6 +50,14 @@ def preprocessing_fn(example: Dict) -> Dict[str, str]:
 
 _ALLOWED_RESPONSE_KEYS = {'response', 'completion'}
 _ALLOWED_PROMPT_KEYS = {'prompt'}
+_DOWNLOADED_FT_DATASETS_DIRPATH = os.path.join(
```

Comment:
instead of chained `dirname`, can you use `os.pardir` (I know I wrote it originally, but lets fix it :P)?